Makefile

```diff
@@ -48,6 +48,7 @@ TARGETS = \
 		nvidia-container-toolkit \
 		nvidia-fabricmanager \
 		nvidia-open-gpu-kernel-modules
+# mellanox-ofed \
```

Comment:
move it down under the comment for drbd?

Reply:
fixed